mono/utils/mono-threads-state-machine.c

```diff
@@ -27,13 +27,20 @@ get_thread_suspend_count (int thread_state)
 	return (thread_state & THREAD_SUSPEND_COUNT_MASK) >> THREAD_SUSPEND_COUNT_SHIFT;
 }
 
+static inline gboolean
+get_thread_wont_safept (int thread_state)
+{
+	return (thread_state & THREAD_SUSPEND_WONT_SAFEPT_MASK) >> THREAD_SUSPEND_WONT_SAFEPT_SHIFT;
```

Comment:
nits: `wont` and `safept` should be spelled out, and seems like a good place for a bitfield.